client/blocks/importer/wordpress/import-everything/pre-migration/index.tsx

```diff
@@ -135,9 +138,10 @@ export const PreMigrationScreen: React.FunctionComponent< PreMigrationProps > =
 		fetchMigrationEnabledStatus();
 	};
 
-	const onFreeTrialClick = () => {
-		addHostingTrial( targetSite.ID, PLAN_MIGRATION_TRIAL_MONTHLY );
-	};
+	// Initiate the migration if initImportRun is set
+	useEffect( () => {
+		initImportRun && startImport( { type: 'without-credentials' } );
+	}, [] );
```

Comment:
Should we also remove this bit?

Reply:
This feature is not in use right now. If we decide in the meantime to include it, it will be accessible by providing a `run` query param. So I'm leaving it as it is.